### ISSUE TITLE
Electron 5.0.1 breaks TypeORM, works with 4.0.1

### ISSUE THREAD

**Issue by psyklopz** [typeorm/typeorm#4111]
**Issue type:**

[x] question
[ ] bug report
[ ] feature request
[ ] documentation issue

**Database system/driver:**

[ ] `cordova`
[ ] `mongodb`
[ ] `mssql`
[x] `mysql` / `mariadb`
[ ] `oracle`
[ ] `postgres`
[ ] `cockroachdb`
[ ] `sqlite`
[ ] `sqljs`
[ ] `react-native`
[ ] `expo`

**TypeORM version:**

[x] `latest`
[ ] `@next`
[ ] `0.x.x` (or put your version here)

**Steps to reproduce or a small repository showing the problem:**

Forgive me for filing this report without all the details.  I'm traveling and I need to make my trip home.  I will fill in the details later.

I had two Electron projects, one 5.0.1 and one 4.0.1.  The data model in the 5.0 version was copied from the 4.0 version.  I was getting `EntityMetadataNotFound: No metadata for "User" was found.` exception on the 5.0 version, but once downgrading Electron to 4.0.1 the problem resolved itself.

Please be aware that changing the electron version in your `package.json` and re-running `npm install` seems to leave the old electron executable.  You must delete `node_modules` before re-installing.

Has anyone else observed trouble with Electron 5.0.1?

**Comment by Kononnable** [comment]
@psyklopz 	I guess you've made your way home by now :)
Any more details? Maybe some default configuration was changed with electron major release?

**Comment by psyklopz** [comment]
After a lot of fussing, it looks like this was an incompatibility with Angular 7.2.  Somehow I managed to get two different versions of Typescript installed side-by-side, one pre 3.3 (needed by Angular) and one post 3.3 (needed by TypeORM).  I didn't know this was possible with npm?  Perhaps one was installed globally and one was local?
In any case, their is no overlap between Angular 7.2 and the latest TypeORM's dependency needs.
Not a big deal, I just upgraded to Angular 8.0, and am using Typescript 3.4.2.  Works with both.
Now if I could just get Angular and ElectronJS to play together.  It doesn't like ES6 modules loaded from a file...
